src/main/webapp/app/shared/modal/modify-therapy-modal.store.tsx

```diff
@@ -0,0 +1,61 @@
+import { action, makeObservable, observable } from 'mobx';
+import { DrugSelectOption } from '../select/DrugSelect';
+
+export class ModifyTherapyModalStore {
```

Comment:
I didn't expect this to be a global store, but rather only exists when modal opens. It should be destroyed after closing.

Reply:
Will readdress after listener refactor